### DISCUSSION TITLE
left-sidebar-check.php has a div tag with a class of just "col-md" (no number) ...why?

### DISCUSSION THREAD

**Discussion by currentcreative** [understrap/understrap-child#385] [Q&A] [answered]
Isn't just "col-md" an invalid class? All the Bootstrap documentation I see seems to suggest a number is needed such as "col-md-8" or "col-md-12" etc.

**Comment by bryanwaddington** [comment, marked as answer]
col-md is a valid class. Without the number, equal columns are created from that breakpoint and up. In the following example there are three equal columns on small and four on medium. On small the fourth column wraps to the next row.

[https://codepen.io/bryanwadd/pen/dyajMev](https://codepen.io/bryanwadd/pen/dyajMev)

```
<div class="container">
  <div class="row">
    <div class="col-sm card">
      One of three columns
    </div>
    <div class="col-sm card">
      One of three columns
    </div>
    <div class="col-sm card">
      One of three columns
    </div>
    <div class="col-md card">
      One of three columns
    </div>
  </div>
</div>
```

**Reply by bacoords** [reply]
Yep - You can also browse their documentation which explains what those columns would be: https://getbootstrap.com/docs/5.3/layout/grid/#responsive-classes

**Reply by currentcreative** [reply]
Man, I am really bad at reading documentation because I was examining that page before I posted this. Thank you both and sorry to post something so inept...

**Reply by bacoords** [reply]
@currentcreative Ha honestly I used bootstrap for years before I knew about those classes.

**Reply by currentcreative** [reply]
Thanks, that makes me feel a little better.

**Reply by bryanwaddington** [reply]
I don't think the documentation is very clear at all regarding col-* without a number so I'm not at all surprised at your question.

Happy to help.
